packages/venia-concept/src/components/ForgotPassword/forgotPassword.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classify from 'src/classify';
+import ForgotPasswordForm from './ForgotPasswordForm';
+import FormSubmissionSuccessful from './FormSubmissionSuccessful';
+import defaultClasses from './forgotPassword.css';
+import { resetPasswordRequest } from './api';
+
+class ForgotPassword extends Component {
+    static propTypes = {
+        classes: PropTypes.shape({
+            instructions: PropTypes.string
+        }),
+        initialValues: PropTypes.shape({
+            email: PropTypes.string
+        }),
+        onClose: PropTypes.func
+    };
+
+    state = {
+        email: '',
```

Comment:
Does it make sense to initialize `state.email` to `props.initialValues.email`, if present?

Reply:
We set email when we change submitSucceeded flag, so we do not need to do this. We use this variable from state only after successful submission of the form.

Reply:
Gotcha.

My concern is that that may be the case today but it opens the door for easy mistakes in the future.

In general if a component keeps track of something in its `state`, we should strive for that `state` to have the correct value as often / as much as possible, to reduce the potential for errors. In this case, we might already know the `email` and so I think we should seed `state.email` with it.

Reply:
I guess I need to change the name of the variable in the state to submittedEmail or something like this.

Reply:
There is no sense to syncronize email, entered in the form, with the state. This field should only store the value, that was submitted by the user

Reply:
This field means something like forgotPasswordRequestPayload.

Reply:
Okay, so you're saying the answer to

> Does it make sense to initialize state.email to props.initialValues.email, if present?

is "no" ?

I was legitimately not sure if these two things were the same or not. Changing the name of one or both of them could definitely help the confusion there 👍